DOM/js/script.js

```diff
@@ -0,0 +1,5 @@
+var elClass = document.getElementsByClassName('button');
+
+for (var i = 0; i <= elClass.length -1; i++) {
```

Comment:
prostsza i bardziej przejrzysta konstrukcja bylaby taka:
`for (var i = 0; i < elClass.length; i++) {`